### PR TITLE
Updates to android setForegroundOptions docs

### PR DESCRIPTION
**Pull request by KennyHuRadar**
## What?

Code snippet of implementing the `Radar.setForegroundOptions` and also updates to the fields it takes. 
## Why?

An additional field, `iconString` was added so that it would be easier to determine the android asset used in the foreground service notification. 
## How?

- Created tabs for android and react native code snippets
- Added a bullet point to explain new field.  

## Screenshots (optional)

image
image

### CODE REVIEW

File: docs/sdk/tracking.mdx [hunk 1]
```diff
@@ -143,9 +143,45 @@ Option | Efficient | Responsive | Continuous
...
+```
+
+  </TabItem>
+
+</Tabs>
+
 - **`text`**: Determines the notification text. Defaults to `"Location tracking started"`.
 - **`title`**: Determines the notification title. Optional.
-- **`icon`**: Determines the notification icon, like `R.drawable.ic_your_icon`. Optional, defaults to `applicationContext.applicationInfo.icon`.
+- **`icon`**: **Deprecated** Determines the notification icon, like `2131558400` (Android's identifier for the resource). Optional, defaults to `iconString`.
```
Comment (jsani-radar) [inline docs/sdk/tracking.mdx:183]: where does this number come from? curious how customers were mapping resources to numbers in the past

File: docs/sdk/tracking.mdx [hunk 2]
```diff
@@ -143,9 +143,45 @@ Option | Efficient | Responsive | Continuous
 
 To avoid [Background Location Limits](https://developer.android.com/about/versions/oreo/background-location-limits), you can start a foreground service and show a notification during tracking. If using Android SDK < [3.4.1](https://github.com/radarlabs/radar-sdk-android/releases/tag/3.1.9), set `trackingOptions.foregroundService` to an instance of `RadarTrackingOptionsForegroundService`. If using Android SDK >= [3.4.1](https://github.com/radarlabs/radar-sdk-android/blob/master/MIGRATION.md#33x-to-34x), use `Radar.setForegroundServiceOptions()` to customize the foreground service notification appearance passing in an instance of `RadarTrackingOptionsForegroundService`.
 
+<Tabs
+  groupId="setForegroundServiceOptions"
+  defaultValue="kotlin"
+  values={[
+    { label: 'Kotlin', value: 'kotlin' },
+    { label: 'React Native', value: 'reactNative' },
```
Comment (jsani-radar) [inline docs/sdk/tracking.mdx:151]: value should be `react-native` for consistency here -https://github.com/radarlabs/documentation/blob/main/docs/fraud.mdx#L251

File: docs/sdk/tracking.mdx [hunk 3]
```diff
@@ -143,9 +143,45 @@ Option | Efficient | Responsive | Continuous
...
+  ]}
+>
+  <TabItem value="kotlin">
+
+```kotlin
+Radar.setForegroundServiceOptions(RadarTrackingOptions.RadarTrackingOptionsForegroundService(
+  text="Text",
+  title="Title",
+  iconString="your_icon_name",
+  updatesOnly = true
```
Comment (jsani-radar) [inline docs/sdk/tracking.mdx:161]: nit: remove spaces before / after `=` here for consistency

**Review by jsani-radar — COMMENTED** [review]
few small comments - overall looking good

File: docs/sdk/tracking.mdx (hunk 1 above)
Comment (KennyHuRadar) [reply docs/sdk/tracking.mdx:183]: I believe that under the hood, android also uses some numbers to identify assets. This particular example is the number that android use to identify the waypoint icon I got from logging statements. I'm not super sure how other dev are doing it in the past

**Review by jsani-radar — APPROVED** [review]

**Pull request merged**